rust/private/rustc.bzl

```diff
@@ -63,6 +63,11 @@ ExtraExecRustcFlagsInfo = provider(
     fields = {"extra_exec_rustc_flags": "List[string] Extra flags to pass to rustc in exec configuration"},
 )
 
+PerCrateRustcFlagsInfo = provider(
+    doc = "Pass each value as an additional flag to non-exec rustc invocations for crates matching the provided filter",
+    fields = {"per_crate_rustc_flags": "List[string] Extra flags to pass to rustc in non-exec configuration"},
```

Comment:
nit, this could probably be shortened to `flags`

Reply:
Agreed, but I chose this for consistency with the other providers defined in this file, which use this naming convention. What do you think about being the change we want to see in the world vs stylistic consistency?